src/overlays/actors/ovl_Obj_Mu_Pict/z_obj_mu_pict.c

```diff
@@ -21,30 +30,164 @@ const ActorInit Obj_Mu_Pict_InitVars = {
     (ActorFunc)ObjMuPict_Update,
     (ActorFunc)ObjMuPict_Draw,
 };
-*/
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/ObjMuPict_Init.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/ObjMuPict_Destroy.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06B5C.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06B70.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06C54.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06C68.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06CC4.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06CD8.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06D90.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06DC8.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/func_80C06E88.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/ObjMuPict_Update.asm")
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Mu_Pict_0x80C06AA0/ObjMuPict_Draw.asm")
+void ObjMuPict_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjMuPict* this = THIS;
+
+    if ((gSaveContext.weekEventReg[75] & 0x20) == 0) {
```

Comment:
I would personally prefer to see stuff like `if (!(gSaveContext.weekEventReg[75] & 0x20))` if it matches still (not sure what the general consensus is though)

Reply:
I would say yes since that's what we've been doing for other cases, but it's certainly open to discussion

Reply:
OOT always does a check against 0. I'm personally not picky. I don't mind it being written either way.